### ISSUE TITLE
Pin Graph Nodes and Edges

### ISSUE THREAD

**Issue by wrongway88** [CoatiSoftware/Sourcetrail#50]
Pin graph nodes and edges to the screen so they don't disappear when moving on to another node. Would enable creating a specific map of a certain procedure, kinda like the attached image

![Image](https://dl.dropboxusercontent.com/u/17330041/Coati/IMG_20160308_160140.jpg)

**Comment by payload** [comment]
+1

**Comment by egraether** [comment]
+1 by mail

**Comment by egraether** [comment]
Better implemented as 'trailing mode' #589

**Comment by PrototypeNM1** [comment]
Pinning has an orthogonal use case not covered by trailing mode: building a map instead of a breadcrumb trail. If you want to build up a code map to augment a mental map, extending the map into dead ends or resetting it on search works against the user's needs.

**Comment by mlangkabel** [comment]
That's true. Pinning more than one node would not be covered by 'trailing mode'.
